lib/quby/answers/services/answer_validator.rb

```diff
@@ -102,7 +136,7 @@ def validate_minimum(question, validation, value)
         end
 
         def validate_maximum(question, validation, value)
-          return if value.blank?
+          return if value.blank? || (question.type == :date && value.values.all?(&:empty?))
```

Comment:
Gebruik liever `blank?`, deze is aan alle objecten toegevoegd door Rails. `empty?` is specifiek voor enumerables geloof ik.